packages/block-editor/src/components/global-styles/background-panel.js

```diff
@@ -318,8 +318,12 @@ function BackgroundImageControls( {
 			return;
 		}
 
-		const sizeValue = style?.background?.backgroundSize;
-		const positionValue = style?.background?.backgroundPosition;
+		const sizeValue =
+			style?.background?.backgroundSize ||
+			inheritedValue?.background?.backgroundSize;
+		const positionValue =
+			style?.background?.backgroundPosition ||
+			inheritedValue?.background?.backgroundPosition;
```

Comment:
This ensures inherited values (from global styles) are taken into account.

Reply:
Whoops, I think I introduced a bug here. We don't want the inherited styles to creep into the block supports styles - see the `onChange` below.

Note to self for tomorrow:

To test, insert a group block that has global styles. Replace the image with an uploaded one. 

The styles from global styles should apply, but not be saved to the inline CSS.

The easiest thing to do would be to reset the styles once an image is uploaded, but that would ignore any theme styles. 🤔

Reply:
WIP https://github.com/WordPress/gutenberg/pull/64328